module_3/src/components/header/MovieDetails.js

```diff
@@ -0,0 +1,27 @@
+import './MovieDetails.css';
+
+import React from "react";
+
+export default function MovieDetails(props) {
```

Comment:
If you want, you can destructure the props here, and even rename it to m:
```suggestion
export default function MovieDetails({ movie: m }) {
```